assignments/can_drive/can_drive.js

```diff
@@ -1,5 +1,20 @@
 function CanDrive(hasDrivingLiscence, isTired, isSober) {
     // Write you code here
+    if (hasDrivingLiscence){
+        
+        if(isTired==true){
+            return "You shouldn't drive"
+        }
```

Comment:
you can combine these 2 conditions as they are returning same value

Reply:
yes sir